### PR TITLE
migrated navbar over

### PR DESCRIPTION
**Pull request by styled-man**

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
-  children,
-}: {
-  children: React.ReactNode
-}) {
+export default function RootLayout({ children }: LayoutProps) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={inter.className}>
+        <header>
```
Comment (raidel-a) [inline src/app/layout.tsx:21]: redundant

File: src/components/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    <nav className="grid p-8 max-w-screen-2xl mx-auto md:grid-cols-2 lg:grid-cols-3 ">
+      <div className="relative mb-7">
+        <Link href="/">
+          <Image
+            className="w-20"
+            src="/assets/images/logo.svg"
+            alt="Logo"
+            width={300}
+            height={120}
+            sizes="100vw"
```
Comment (raidel-a) [inline src/components/Navbar.tsx:69]: redundant

File: public/assets/images/logo.svg [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<svg
+    className="h-6 cursor-pointer self-center whitespace-nowrap text-white  hover:text-primary_yellow"
+    fill="currentColor"
+    strokeWidth="0"
+    id="Footer INIT Logo"
```
Comment (raidel-a) [inline public/assets/images/logo.svg:5]: might want to properly label these

File: src/components/Navbar.tsx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    {
+      name: "Linkedin",
+      href: "https://www.linkedin.com/company/initofficial/",
+      Icon: FaLinkedinIn,
+      style: "bg-blue-600 hover:bg-blue-700",
+    },
+  ];
+
+  return (
+    <nav className="grid p-8 max-w-screen-2xl mx-auto md:grid-cols-2 lg:grid-cols-3 ">
```
Comment (raidel-a) [inline src/components/Navbar.tsx:60]: nav links disappear before menu icon is visible.
quick and dirty fix could be ...
    <nav className="grid p-8 max-w-screen-2xl mx-auto  md:grid-cols-3 ">

File: src/components/Navbar.tsx [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+        <button
+          className="absolute top-0 right-0 md:hidden"
+          onClick={() => setIsOpen(!isOpen)}
+        >
+          <IoMenu className="text-3xl" />
+        </button>
+      </div>
+
+      <div className={twMerge(!isOpen && "hidden", "mb-7 md:hidden lg:block")}>
```
Comment (raidel-a) [inline src/components/Navbar.tsx:81]: nav links disappear before menu icon is visible.
quick and dirty fix could be ...
 <div className={twMerge(!isOpen && "hidden", "mb-7  md:block")}>
        <ul className="flex flex-col relative md:-left-20  md:flex-row items-start 
			lg:items-center justify-evenly md:gap-2 ">

**Review by raidel-a — CHANGES_REQUESTED** [review]
Navigation links disappear before the menu icon appears due to the class grid-cols-2 being applied at the medium breakpoint. Found and suggested a hotfix, but we should find a better way to handle this issue.

File: src/components/Navbar.tsx (hunk 2 above)
Comment (styled-man) [reply src/components/Navbar.tsx:69]: The `alt`, `width` and `height` props are required
the `size` prop is to ensure that it behaves predictably for all screen sizes.

Comment (raidel-a) [reply src/components/Navbar.tsx:69]: I thought it defaulted to 100vw, good to know.

File: src/components/Navbar.tsx
Comment (02danieljonas) [inline src/components/Navbar.tsx:1]: I think the navbar should be consistent with every element having an onhover

**Review by 02danieljonas — APPROVED** [review]
Other than my comments, LGTM!!!

**Review by raidel-a — APPROVED** [review]
👍 LGTM

**Pull request merged**